hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlSessions.java

```diff
@@ -312,12 +315,9 @@ public Integer commit() {
                 }
                 this.conn.commit();
                 this.clear();
+                this.end();
```

Comment:
How to deal with the failure situation of end(), at this time it has been committed successfully, but the upper layer will think the transaction failed, and there is no effect after calling rollback().